### DISCUSSION TITLE
Reading geojson info from map

### DISCUSSION THREAD

**Discussion by cubecleveland** [maplibre/maplibre-gl-js#766] [Q&A] [answered]
Hello,
We have an app that is enabling drawing shapes uding the draw functionallity, we want to then save the geojson object toa  dtabase.
any example to show how we can loop the drawn objects on the map ?

**Comment by wipfli** [comment, marked as answer]
https://maplibre.org/maplibre-gl-js-docs/example/mapbox-gl-draw/
